draft-terbu-oauth-jwt-vc.md

```diff
@@ -83,27 +83,27 @@ To support revocation of Verifiable Credentials, an optional fourth party can be
 involved, a Status Provider, who delivers revocation information to Verifiers.
 (The Verifier can also serve as the Status Provider.)
 
-This specification defines Verifiable Credentials based on the SD-JWT
-format with a JWT Claim Set.
+This specification defines Verifiable Credentials based on JWT and SD-JWT.
 
 ## Rationale
 
-JSON Web Tokens (JWTs) [@!RFC7519] can in principle be used to express
-Verifiable Credentials in a way that is easy to understand and process as it
-builds upon established web primitives. While JWT-based credentials enable selective
-disclosure, i.e., the ability for a Holder to disclose only a subset of the contained
-claims, in an Identity Provider ecosystem by issuing new JWTs to the Verifier for
-every presentation, this approach does not work in the three-party-model.
+JSON Web Tokens (JWTs) [@!RFC7519] can be used to express
+Verifiable Credentials in the three-party-model
+in a way that is easy to understand and process as it
+builds upon established web primitives.
+
 
 Selective Disclosure JWT (SD-JWT) [@!I-D.ietf-oauth-selective-disclosure-jwt] is
 a specification that introduces conventions to support selective disclosure for
 JWTs: For an SD-JWT document, a Holder can decide which claims to release (within
-bounds defined by the Issuer). This format is therefore perfectly suited for
-Verifiable Credentials.
+bounds defined by the Issuer).
 
 SD-JWT itself does not define the claims that must be used within the payload or
-their semantics. This specification therefore defines how Verifiable Credentials
-can be expressed using SD-JWT.
+their semantics. JWT defines rules for payload but does not define how Verifiable
+Credentials can be expressed using JWT.
+
+This specification therefore defines how Verifiable Credentials
+can be expressed using JWT or SD-JWT.
```

Comment:
Should this be improved:

This specification defines how Verifiable Credentials can be expressed using JWT claim set and protected with JWS or SD-JWT?

(
note that if convergence between JWS and SD-JWT is achieved it could be even something like:

This specification defines how Verifiable Credentials can be expressed using JWT claim set and protected using JWS. The specification also defines how Selective Disclosure (SD-JWT) can be applied.
)